features/Websocket.feature

```diff
@@ -616,9 +616,43 @@ Feature: Test websocket API
     And I create a new role "role2" with id "role2"
     And I create a new profile "profile2" with id "profile2"
     And I create a user "user2" with id "user2-id"
-    When I log in as user2-id:testpwd2 expiring in 1h
+    When I log in as user2:testpwd2 expiring in 1h
     Then I'm able to find my rights
 
+  @usingWebsocket @cleanSecurity
```

Comment:
one is named `user credentials crudl` and the other `current user credentials crudl`. I don't know how to differ them more than that :disappointed: